docs/code_push/initialize.md

```diff
@@ -66,3 +66,7 @@ able to communicate with the Shorebird servers to pull new patches.
 Running `shorebird doctor` will check that your `AndroidManifest.xml` file is
 set up correctly.
 :::
+
+:::tip
+If you already have a `shorebird.yaml` and would like to re-initialize your project, use `shorebird init --force`.
```

Comment:
What will this do?  Will it make new app ids?  If you already have some patches published on your current app id what do you do then?

Reply:
I'll update to elaborate but it'll just overwrite your current `shorebird.yaml` and make new app ids. Your existing apps won't be affected.